client/blocks/conversations/style.scss

```diff
@@ -2,3 +2,35 @@
 	list-style-type: none;
 	margin-left: 0;
 }
+
+.conversations__stream .card.is-compact {
+	padding: 0;
+}
+
+.conversations__stream .reader-post-card.card .reader-post-options-menu {
+	position: absolute;
+		right: -2px;
+		top: -16px;
+}
+
+.conversations__stream .reader-post-card__title {
+	height: 16px * 1.7;
+	overflow: hidden;
+	position: relative;
+	width: calc( 100% - 120px );
+	white-space: nowrap;
+
+	&::after{
+		@include long-content-fade( $size: 10% );
```

Comment:
@blowery it's set to `5%` in CC. Updated both to use `35px` which is about the same as `5%`: https://github.com/Automattic/wp-calypso/pull/17070/commits/5a206de1b5f9871a5c2a4280b23a2ee123c8b824